bin/tag-generator.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+DEFAULT_ALPINE_VERSION = '3.12'
+
+def generate_tags(key, version):
+    update = version.split('.')[1] if (key == '8') else version.split('.')[2]
+    expanded_version = f"{key}u{update}" if (key == '8') else f"{key}.0.{update}"
+
+    al2_tags = [f"{key}", f"{expanded_version}", f"{expanded_version}-al2", f"{key}-al2-full",f"{key}-al2-jdk"]
+    if key == '8':
+        al2_tags.append('latest')
+    print("Tags: " + ", ".join(al2_tags) + "")
+    print("Architectures: amd64, arm64v8")
+    print(f"Directory: {key}/jdk/al2\n")
+
+    for alpine_version in ['3.12', '3.13', '3.14']:
```

Comment:
Can we pull this list up to a top level var?

Reply:
Done.